rasa/core/policies/flow_policy.py

```diff
@@ -695,17 +695,31 @@ def _run_step(
                     and previous_flow_step
                     and current_frame.frame_type == StackFrameType.CORRECTION
                 ):
-                    # TODO: we need to figure out how to actually
-                    #    "undo" the changed slots
                     corrected_slots = tracker.get_slot(CORRECTED_SLOTS_SLOT)
                     if corrected_slots:
-                        self._correct_flow_position(
-                            corrected_slots, previous_flow_step, previous_flow, tracker
-                        )
-                    else:
-                        # TODO: we need to figure out how to actually "undo" the
-                        #    changed slots
-                        pass
+                        if (
+                            tracker.latest_action_name
+                            == "utter_corrected_previous_input"
```

Comment:
I'm not totally comfortable with the idea of having hardcoded action names in the policy. (at a minimum I think we should define a constant like [here](https://github.com/RasaHQ/rasa/blob/83c9d0c4de05da3b0e01383603cd1002dbfd97bb/rasa/shared/core/constants.py#L25) 

But those other actions are all internal, whereas this is an utter action. Is there not a more robust way to know that we just corrected a slot?

Reply:
ya, totally agree with you. I have now used the intent from the latest user message to check if affirm or deny is chosen.
Please let me know if this approach is fine or if there is a better way to do this.

Reply:
unfortunately I'm not sure that will work. We don't rely on "affirm"/"deny" intents with the command generator anymore

Reply:
This currently works with the command generator, but as we have agreed on changing to default affirmations. Do we hold this task until we implement the changes that we discussed today? 
new task: https://rasahq.atlassian.net/browse/ENG-453